packages/patternfly-4/react-charts/src/components/Chart/Chart.js

```diff
@@ -1,21 +1,49 @@
+/* eslint-disable react/require-default-props */
 import React from 'react';
 import PropTypes from 'prop-types';
 import hoistNonReactStatics from 'hoist-non-react-statics';
 import { VictoryChart } from 'victory';
-import { default as ChartTheme } from '../ChartTheme/ChartTheme';
+import { getTheme } from '../ChartTheme/themes/theme-utils';
 
 export const propTypes = {
   /**
    * See TypeScript API docs: https://formidable.com/open-source/victory/docs/victory-chart/
    */
-  '': PropTypes.any
+  '': PropTypes.any,
+  /**
+   * The height props specifies the height the svg viewBox of the chart container. This value should be given as a
+   * number of pixels.
+   *
+   * Because Victory renders responsive containers, the width and height props do not determine the width and
+   * height of the chart in number of pixels, but instead define an aspect ratio for the chart. The exact number of
+   * pixels will depend on the size of the container the chart is rendered into.
+   */
+  height: PropTypes.number,
+  /*
+   * Specifies the theme color; blue (default), green, or multi-color. Overridden by the theme property.
+   */
+  themeColor: PropTypes.string,
+  /*
+   * Specifies the theme variant; 'dark' or 'light' (default). Overridden by the theme property.
+   */
+  themeVariant: PropTypes.string,
+  /**
+   * The width props specifies the width of the svg viewBox of the chart container. This value should be given as a
+   * number of pixels.
+   *
+   * Because Victory renders responsive containers, the width and height props do not determine the width and
+   * height of the chart in number of pixels, but instead define an aspect ratio for the chart. The exact number of
+   * pixels will depend on the size of the container the chart is rendered into.
+   */
+  width: PropTypes.number
 };
 
-const Chart = ({ children, ...props }) => (
-  <VictoryChart theme={ChartTheme.default} {...props}>
+const Chart = ({ children, theme, themeColor, themeVariant, ...rest }) => (
+  <VictoryChart theme={theme || getTheme(themeColor, themeVariant)} {...rest}>
```

Comment:
```suggestion
  <VictoryChart theme={theme} {...rest}>
```

Reply:
Updated all components to set the theme default in this way, but will only comment once here.